etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,323 @@
+        state: enabled
+    enabled: "{{ 'geneve' in (kolla_neutron_ml2_type_drivers + kolla_neutron_ml2_tenant_network_types) and 'network' in group_names }}"
+  # VXLAN
+  - rules:
+      - port: 4789/udp
+        network: "{{ tunnel_net_name }}"
+        state: enabled
+    enabled: "{{ 'vxlan' in (kolla_neutron_ml2_type_drivers + kolla_neutron_ml2_tenant_network_types) and 'network' in group_names }}"
+  # Octavia
+  - rules:
+      - port: 5555/udp
+        network: "{{ octavia_net_name | default() }}"
+        state: enabled
+    enabled: "{{ kolla_enable_octavia | bool and octavia_net_name is not none }}"
+  # DHCP, Was required to create VMs at a customer site, need to work out condition
+  - rules:
+    - port: 67/udp
+      network: "{{ provision_oc_net_name }}"
+      state: enabled
+    enabled: true
+
+###############################################################################
+# Compute firewalld rules
+
+stackhpc_compute_firewalld_rules_template:
+  # GENEVE
+  - rules:
+      - port: 6081/udp
+        network: "{{ tunnel_net_name }}"
+        state: enabled
+    enabled: "{{ ('geneve' in (kolla_neutron_ml2_type_drivers + kolla_neutron_ml2_tenant_network_types)) | bool }}"
+  # VXLAN
+  - rules:
+      - port: 4789/udp
+        network: "{{ tunnel_net_name }}"
+        state: enabled
+    enabled: "{{ ('vxlan' in (kolla_neutron_ml2_type_drivers + kolla_neutron_ml2_tenant_network_types)) | bool }}"
+
+###############################################################################
+# Storage firewalld rules
+
+stackhpc_storage_firewalld_rules_template:
+  # Ceph Prometheus exporter
+  - rules:
+      - port: 9283/tcp
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+    enabled: "{{ kolla_enable_prometheus_ceph_mgr_exporter | default(false) | bool and 'mgrs' in group_names }}"
+  # Ceph
+  - rules:
+      - service: ceph
+        network: "{{ storage_net_name }}"
+        state: enabled
+      - service: ceph
+        network: "{{ storage_mgmt_net_name }}"
+        state: enabled
+      - service: ceph-mon
+        network: "{{ storage_net_name }}"
+        state: "{{ 'enabled' if 'mons' in group_names else 'disabled' }}"
+    enabled: "{{ 'ceph' in group_names }}"
+
+###############################################################################
+# Monitoring firewalld rules
+
+stackhpc_monitoring_firewalld_rules_template: []
+
+###############################################################################
+# Infra VM firewalld rules (Wazuh Manager)
+
+stackhpc_wazuh_manager_infra_vm_firewalld_rules_template:
+  - rules:
+      - port: 1514/tcp
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - port: 1514/udp
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - port: 1515/tcp
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - port: 443/tcp
+        network: "{{ public_net_name }}"
```

Comment:
I don't think we can assume that the dashboard is on the public API network. How about this?
```suggestion
        network: "{{ wazuh_dashboard_net_name | default(provision_oc_net_name) }}"
```